src/tinuous/__main__.py

```diff
@@ -11,6 +11,7 @@
 from pathlib import Path
 import re
 from shutil import rmtree
+from string import Template
```

Comment:
ho ho -- I think I never used that yet another way to format strings in Python.  I only wonder -- why you decided to go for "Template + format" instead of e.g. just making `vars` to also participate in "format"?

Reply:
Because there's no clean way to pull off nested `{format}` templates.

Reply:
not sure I am following on "nested".  What I meant is instead of 
```python
        template2 = Template(path_template).substitute(vars)
        return template2.format_map(self.path_fields())
```
have smth like
```python
       return template2.format_map(dict(vars, **self.path_fields())
```

Reply:
I've changed the implementation to allow using vars via `{format}` syntax.